### ISSUE TITLE
Cannot download debugger in IDEA 2020.2.1 Preview - 404 error

### ISSUE THREAD

**Issue by lancelote** [intellij-rust/intellij-rust#5921]
## Environment

* **IntelliJ Rust plugin version:** 0.3.129.3281-202-nightly
* **Rust toolchain version:** 1.47.0-nightly (6c8927b0c 2020-07-26) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2020.2.1 Preview Ultimate Edition (IU-202.6948.5)
* **Operating system:** macOS 10.15.4

## Problem description

Cannot download debugger in 2020.2.1 IDEA preview - get 404 error

```
2020-08-12 22:08:16,999 [  81886]   WARN - ger.RsDebuggerToolchainService - Can't download debugger 
java.io.IOException: Failed to download 'https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/org/jetbrains/intellij/deps/cidr/lldbfrontend/27ed608ccb4eef97d0434093cf911d8fa15d8c17/lldbfrontend-27ed608ccb4eef97d0434093cf911d8fa15d8c17-mac-x64.tar.gz':
Request failed with status code 404
	at com.intellij.util.download.impl.FileDownloaderImpl.lambda$download$1(FileDownloaderImpl.java:234)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:215)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:194)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:207)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:183)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.intellij.util.io.HttpRequests$HttpStatusException: Request failed with status code 404. Status=404, Url=https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/org/jetbrains/intellij/deps/cidr/lldbfrontend/27ed608ccb4eef97d0434093cf911d8fa15d8c17/lldbfrontend-27ed608ccb4eef97d0434093cf911d8fa15d8c17-mac-x64.tar.gz
	at com.intellij.util.io.HttpRequests.throwHttpStatusError(HttpRequests.java:644)
	at com.intellij.util.io.HttpRequests.openConnection(HttpRequests.java:625)
	at com.intellij.util.io.HttpRequests.access$300(HttpRequests.java:57)
	at com.intellij.util.io.HttpRequests$RequestImpl.getConnection(HttpRequests.java:376)
	at com.intellij.util.download.impl.FileDownloaderImpl$2.process(FileDownloaderImpl.java:337)
	at com.intellij.util.download.impl.FileDownloaderImpl$2.process(FileDownloaderImpl.java:334)
	at com.intellij.util.io.HttpRequests.doProcess(HttpRequests.java:517)
	at com.intellij.util.io.HttpRequests.process(HttpRequests.java:499)
	at com.intellij.util.io.HttpRequests.access$100(HttpRequests.java:57)
	at com.intellij.util.io.HttpRequests$RequestBuilderImpl.connect(HttpRequests.java:350)
	at com.intellij.util.download.impl.FileDownloaderImpl.downloadFile(FileDownloaderImpl.java:334)
	at com.intellij.util.download.impl.FileDownloaderImpl.lambda$download$1(FileDownloaderImpl.java:231)
	... 13 more
```

**Comment by lancelote** [comment]
https://youtrack.jetbrains.com/issue/IDEA-248114

**Comment by Undin** [comment]
@lancelote check it again, please. Should be fixed now

**Comment by lancelote** [comment]
Works fine, thank you 👍

**Comment by Undin** [comment]
@sayyiedmo2 Could you create a separate issue, please? New comments to issue which was closed 2.5 years ago without any additional info are not very useful. `Help | Create New Issue` action may help you
